README.md

```diff
@@ -11,11 +11,34 @@ We encourage prospective users to try out the demo, and use the provided
 [manifests](two_macvtap_fedora_vms.yaml) as reference for more advanced
 use cases.
 
+# Installation
+To be able to run the demo, you'll need to clone the following repos:
+  - [kubevirt](https://github.com/kubevirt/kubevirt/):
+    `git clone https://github.com/maiqueb/kubevirt/ -branch add_macvtap_binding_mech`
```

Comment:
would it be better to fetch the PR to the original kubevirt repo instead ?

Reply:
No, I think is fine as it is. It's hopefully temporary anyway.